symbol-overlay.el

```diff
@@ -336,7 +342,10 @@ Otherwise apply `symbol-overlay-default-face'."
     (if face (progn (overlay-put ov 'face face)
                     (overlay-put ov 'keymap symbol-overlay-map)
                     (overlay-put ov 'evaporate t)
-                    (overlay-put ov 'symbol symbol))
+                    (overlay-put ov 'symbol symbol)
+                    (overlay-put ov
+                                 'cursor-sensor-functions
+                                 `(,symbol-overlay-cursor-hook)))
```

Comment:
It's a bit of a red flag that this is setting an array value using a hard-coded list of one value. What if there were existing `cursor-sensor-functions` on the underlying text?